src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -771,22 +771,27 @@ namespace FourPipeBeam {
 				this->vDotDesignHW = this->vDotNormRatedHW * this->totBeamLength;
 			}
 		}
-		// fill in mass flow rate versions of working variables (regardless of autosizing)
+		// fill in mass flow rate versions of working variables (regardless of autosizing )
 		this->mDotDesignPrimAir = this->vDotDesignPrimAir * DataEnvironment::StdRhoAir;
 
 		if ( ( originalTermUnitSizeMaxVDot > 0.0 ) && ( originalTermUnitSizeMaxVDot != this->vDotDesignPrimAir ) && ( CurZoneEqNum > 0 ) ) {
-			// perturb system size to handle chnage in system size calculated without knowing about 4 pipe beam
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesMainVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeMaxVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesCoolVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeCoolVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).DesHeatVolFlow
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeHeatVDot );
-			DataSizing::FinalSysSizing( DataZoneEquipment::ZoneEquipConfig(CurZoneEqNum).AirLoopNum ).MassFlowAtCoolPeak
-				+= ( this->vDotDesignPrimAir - originalTermUnitSizeCoolVDot ) * DataEnvironment::StdRhoAir;
-
-			ReportSizingOutput( this->unitType, this->name, "AirLoopHVAC Design Supply Air Flow Rate Adjustment [m3/s]",
-								( this->vDotDesignPrimAir - originalTermUnitSizeMaxVDot ) );
+			if ( DataSizing::FinalSysSizing.size() > 0 ) {
```

Comment:
I think you could get this from `DataSizing::SysSizingRunDone`.  @mjwitte that makes sense, right?

I'd **like** to consider it more robust in case those arrays ever get restructured.  In reality, it's probably of equal robustness, but from a code readability standpoint I'd prefer checking the flag.